data/globalevents/scripts/serversave.lua

```diff
@@ -1,14 +1,19 @@
 local function ServerSave()
```

Comment:
the function that actually saves the server is `saveServer()`
it doesn't appear anywhere in this function, therefore the server doesn't really get saved